test/over_react_test/custom_matchers_test.dart

```diff
@@ -562,15 +577,17 @@ main() {
               );
             });
 
-            test('simple usage with error config', () {
-              shouldPass(
-                  () => mount((Sample()..shouldAlwaysBeFalse = true)()),
-                  logsToConsole([
-                    contains('shouldNeverBeNull is required'),
-                    contains('shouldAlwaysBeFalse set to true'),
-                  ], consoleConfig: errorConfig)
-              );
-            });
+            if (runtimeSupportsPropTypeWarnings()) {
+              test('simple usage with error config', () {
+                shouldPass(
+                    () => mount((Sample()..shouldAlwaysBeFalse = true)()),
+                    logsToConsole([
+                      contains('shouldNeverBeNull is required'),
+                      contains('shouldAlwaysBeFalse set to true'),
+                    ], consoleConfig: errorConfig)
+                );
+              });
+            }
```

Comment:
#nit similar to the comment above, since this test isn't about `propTypes` specifically it could be valid to set the component to trigger non-`propTypes` errors and check for those instead

Reply:
Addressed in https://github.com/Workiva/over_react_test/pull/104/commits/5e736cd83aa9091f0e0dd699c9e3d8020ce7fd4f